DorisProject/src/app/pages/stock/stock.component.ts

```diff
@@ -70,4 +73,20 @@ export class StockComponent implements OnInit {
   }
 
 
+  openModal(request: string) {
+    const dialogRef = this.dialog.open(ModalComponent, {data: {textrequest: request, textresponse:this.responseModal}});
```

Comment:
Codacy has a fix for the issue: [Replace `data:·{textrequest:·request,·textresponse:this.responseModal}` with `⏎······data:·{·textrequest:·request,·textresponse:·this.responseModal·},⏎····`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9258681)
```suggestion
    const dialogRef = this.dialog.open(ModalComponent, {
```